src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -90,15 +90,21 @@ public static List<PartyDetails> buildHearingPartiesDetails(SscsCaseData caseDat
                 buildHearingPartiesPartyDetails(
                     caseData.getJointParty(),
                     appellant.getId(),
-                    referenceDataServiceHolder
+                    referenceDataServiceHolder,
+                    null
                 ));
         }
 
         OverrideFields overrideFields = OverridesMapping.getOverrideFields(caseData);
 
+        String adjournLanguage = null;
+        if (isYes(caseData.getAdjournCaseInterpreterRequired())) {
+            adjournLanguage = caseData.getAdjournCaseInterpreterLanguage().getValue().getCode();
+        }
+
```

Comment:
I would doing something along the lines of:
```suggestion
        String adjournLanguage = optional.ofNullable(caseData)
            .filter(case -> isYes(case .getAdjournCaseInterpreterRequired())
            .map(SscsCaseData::getAdjournCaseInterpreterLanguage)
            .map(DynamicList::getValue)
            .map(DynamicList::getCode)
            .filter(StringUtils::isNotBlank)
            .orElse(null)
```